cmd/msrc/generate.go

```diff
@@ -0,0 +1,146 @@
+package main
```

Comment:
just for my own understanding: who's going to call this command?

Reply:
Sorry I prob should've mentioned it - I'll be updating [this](https://github.com/fleetdm/nvd/blob/master/.github/workflows/generate.yml) in https://github.com/fleetdm/fleet/issues/7493

Reply:
makes sense, thank you!